src/renderer/views/Channel/Channel.js

```diff
@@ -1048,7 +1048,7 @@ export default defineComponent({
       }
     },
 
-    getChannelInfoInvidious: function () {
+    getChannelInfoInvidious: async function () {
```

Comment:
This function doesn't use await, so it could probably be switched back to a normal sync function.